README.md

```diff
@@ -1,6 +1,6 @@
-# Octopus People
+﻿# Octopus People
 
```

Comment:
Should this be "as Octopus", or "at Octopus"?

Reply:
I agree: "at Octopus". :)